Crash when importing wrong playlist format

### Steps to reproduce

1. Import Playlist in the Settings
2. App Crashes

### Expected behavior

Show the Playlist

### Actual behavior

The App crashes

### LibreTube version

0.9.0

### Android version

Android 12

### Other details

java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1252759818, result=-1, data=Intent { dat=content://com.android.externalstorage.documents/document/CADE-15F5:Documents/Backups/Google/YouTube/Takeout/YouTube e YouTube Music/playlists/Piano Music.csv flg=0x1 }} to activity {com.github.libretube/com.github.libretube.ui.activities.SettingsActivity}: com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'Código': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')
 at [Source: (String)"Código da playlist,ID do canal,Horário da criação,Horário da atualização,Título,Descrição,Visibilidade
PLFYR_my6zxpXUiPHebMGxk9Vch2g0tcKk,UCAZJYKEdxjKG3AH0dx0ZGhw,2018-06-15 01:12:22 UTC,2022-11-09 19:18:25 UTC,Piano Music,,Não listado

ID do vídeo,Horário da adição
sEQf5lcnj_o,2019-04-11 19:52:52 UTC
nHONksx5R_0,2018-06-15 01:20:20 UTC
O8s4OcRJc3w,2019-01-02 23:25:37 UTC
luXjNItbHC4,2018-10-28 17:47:29 UTC
A1GK6e52iss,2020-02-02 02:20:02 UTC
w40eGwOoJz0,2022-04-25 19:30:48 UTC
5Ggnzs2hP3s,2022-"[truncated 4701 chars]; line: 1, column: 7]
	at android.app.ActivityThread.deliverResults(ActivityThread.java:5324)
	at android.app.ActivityThread.handleSendResult(ActivityThread.java:5363)
	at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:67)
	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2242)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7833)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
Caused by: com.fasterxml.jackson.core.JsonParseException: Unrecognized token 'Código': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')
 at [Source: (String)"Código da playlist,ID do canal,Horário da criação,Horário da atualização,Título,Descrição,Visibilidade
PLFYR_my6zxpXUiPHebMGxk9Vch2g0tcKk,UCAZJYKEdxjKG3AH0dx0ZGhw,2018-06-15 01:12:22 UTC,2022-11-09 19:18:25 UTC,Piano Music,,Não listado

ID do vídeo,Horário da adição
sEQf5lcnj_o,2019-04-11 19:52:52 UTC
nHONksx5R_0,2018-06-15 01:20:20 UTC
O8s4OcRJc3w,2019-01-02 23:25:37 UTC
luXjNItbHC4,2018-10-28 17:47:29 UTC
A1GK6e52iss,2020-02-02 02:20:02 UTC
w40eGwOoJz0,2022-04-25 19:30:48 UTC
5Ggnzs2hP3s,2022-"[truncated 4701 chars]; line: 1, column: 7]
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._reportInvalidToken(ReaderBasedJsonParser.java:69)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._handleOddValue(ReaderBasedJsonParser.java:275)
	at com.fasterxml.jackson.core.json.ReaderBasedJsonParser.nextToken(ReaderBasedJsonParser.java:169)
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:42)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:15)
	at com.github.libretube.ui.preferences.BackupRestoreSettings$$ExternalSyntheticLambda4.onActivityResult(R8$$SyntheticClass:35)
	at androidx.activity.result.ActivityResultRegistry.dispatchResult(ActivityResultRegistry.java:47)
	at androidx.activity.ComponentActivity.onActivityResult(ComponentActivity.java:3)
	at androidx.fragment.app.FragmentActivity.onActivityResult(FragmentActivity.java:6)
	at android.app.Activity.dispatchActivityResult(Activity.java:8388)
	at android.app.ActivityThread.deliverResults(ActivityThread.java:5317)
	... 13 more
    

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

--- a/app/src/main/java/com/github/libretube/util/ImportHelper.kt
+++ b/app/src/main/java/com/github/libretube/util/ImportHelper.kt
@@ -115,8 +115,15 @@ class ImportHelper(
         val playlistFile = ObjectMapper().readValue(uri.readText(), ImportPlaylistFile::class.java)
 
         CoroutineScope(Dispatchers.IO).launch {
-            playlistFile.playlists?.let {
-                PlaylistsHelper.importPlaylists(activity, it)
+            try {
+                playlistFile.playlists?.let {
+                    PlaylistsHelper.importPlaylists(activity, it)
+                }
+            } catch (e: Exception) {
+                Log.e(TAG(), e.toString())
+                e.localizedMessage?.let {
+                    activity.applicationContext.toastFromMainThread(it)
+                }
             }
         }
 

